Casks/a/algodoo.rb

```diff
@@ -1,15 +1,15 @@
 cask "algodoo" do
-  version "2.1.3"
-  sha256 "9f3419d0da9cca0f0f5abc0b8a228197221c92fcbc35138ed2bb0b9251820a66"
+  version "2.2.1"
+  sha256 "67bad62930161e38356ceeb1e4fa8dd0b28eb234fa0aaa14ca927766808330ce"
 
-  url "https://www.algodoo.com/download/Algodoo_#{version.dots_to_underscores}-MacOS.dmg"
+  url "https://www.algodoo.com/download/Algodoo_#{version.dots_to_underscores}-macOS.dmg"
   name "Algodoo"
   desc "Draw and interact with physical systems"
   homepage "https://www.algodoo.com/"
 
   livecheck do
     url "https://www.algodoo.com/download/"
-    regex(%r{href=.*?/Algodoo_(\d+(?:_\d+)*)-MacOS\.dmg}i)
+    regex(%r{href=.*?/Algodoo[._-](\d+(?:_\d+)*)[._-]MacOS\.dmg}i)
```

Comment:
```suggestion
    regex(/href=.*?Algodoo[._-]v?(\d+(?:[._]\d+)+)[._-]macOS\.dmg/i)
```

I overlooked this when I updated the `livecheck` block URL the other day but let's bring this in line with typical patterns while we're at it.

Reply:
You got it! Thanks for the call out!